skbeam/core/accumulators/binned_statistic.py

```diff
@@ -572,6 +572,9 @@ def __init__(self, shape, bins=10, range=None,
         mask : 2-dimensional np.ndarray of ints, optional
             array of zero/non-zero values, with shape `shape`.
             zero values will be ignored.
+        r_map : the map of pixel radii for each pixel. This is useful when the
```

Comment:
add line to explain data type of r_map, like 2-dimensional np.ndarray, optional?

Reply:
r_map can be used to define the radius of each pixel relative to the origin in reciprocal space, like Ewald sphere?

Reply:
thanks. and yes, I like your wording better. I have added it in if that's okay. thanks!